src/hugchat/cli.py

```diff
@@ -88,6 +92,17 @@ def handle_command(chatbot: ChatBot, userInput: str) -> None:
         except Exception as e:
             print(f"Error: {e}")
 
+    elif command == "prompt":
+        try:
+            if len(userInput) > 7:
+                id = int(userInput.split(' ')[1])
```

Comment:
We already have a variable defined (`arguments`) which allows us to get this data without having to split the original string again. Try `arguments[0]`